data/places.ts

```diff
@@ -0,0 +1,37 @@
+/* This is just a dummy file to test that the db connection works 
+The content of this file should be changed later when we start writing database functions
+*/
+
+import { mongoCollections } from "../mongoCollections";
+const { places } = mongoCollections;
+import { ObjectId } from "mongodb";
+
+export async function getAllPlaces() {
+	const placesCollection = await places();
+
+	return await placesCollection.find({}).toArray();
+}
+
+export async function getPlaceByID(id: ObjectId) {
+	const placesCollection = await places();
+
+	const place = await placesCollection.findOne({ _id: id });
+	if (place === null) throw "Sorry, no place exists with that ID";
+	return place;
+}
+
+export async function addPlace(name: string, establishmentType: string) {
```

Comment:
I think we should make most of our create/update functions take an object instead of a list of arguments. that way you wind up with `addPlace(place: Place)` and we can create one reusable type for Place. Plus when the functions are used, you can "build" an object to pass in, instead of destructuring an object just to pass each field back in a list - i imagine eventually we'll have documents with a lot more fields than this!

Reply:
I like this a lot!!! Gosh code review is such a great thing, huh! Learned a ton of things based on how you and @carpenterd777 think! whoop whoop!